inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/exporter/RecommenderExporter.java

```diff
@@ -0,0 +1,120 @@
+    }
+
+    @Override
+    public List<Class<? extends ProjectExporter>> getImportDependencies() {
+        return asList(LayerExporter.class);
+    }
+
+    @Override
+    public List<Class<? extends ProjectExporter>> getExportDependencies() {
+        return asList(LayerExporter.class);
+    }
+
+    @Override
+    public void exportData(ProjectExportRequest aRequest, ExportedProject aExProject, File aFile) {
+        Project project = aRequest.getProject();
+
+        List<ExportedRecommender> exportedRecommenders = new ArrayList<>();
+        for (Recommender recommender : recommendationService.listRecommenders(project)) {
+            ExportedRecommender exportedRecommender = new ExportedRecommender();
+            exportedRecommender.setAlwaysSelected(recommender.isAlwaysSelected());
+            exportedRecommender.setFeature(recommender.getFeature());
+            exportedRecommender.setEnabled(recommender.isEnabled());
+            exportedRecommender.setLayerName(recommender.getLayer().getName());
+            exportedRecommender.setName(recommender.getName());
+            exportedRecommender.setThreshold(recommender.getThreshold());
+            exportedRecommender.setTool(recommender.getTool());
+            exportedRecommender.setTraits(recommender.getTraits());
+            exportedRecommenders.add(exportedRecommender);
+        }
+
+        aExProject.setProperty(KEY, exportedRecommenders);
+        int n = exportedRecommenders.size();
+        LOG.info("Exported [{}] recommender for project [{}]", n, project.getName());
+    }
+
+    @Override
+    public void importData(ProjectImportRequest aRequest, Project aProject,
+                           ExportedProject aExProject, ZipFile aZip) {
+        ExportedRecommender[] recommenders = aExProject
+                .getArrayProperty(KEY, ExportedRecommender.class);
+
+        for (ExportedRecommender exportedRecommender : recommenders) {
+            Recommender recommender = new Recommender();
+            recommender.setAlwaysSelected(exportedRecommender.isAlwaysSelected());
+            recommender.setFeature(exportedRecommender.getFeature());
+            recommender.setEnabled(exportedRecommender.isEnabled());
+            recommender.setName(exportedRecommender.getName());
+            recommender.setThreshold(exportedRecommender.getThreshold());
+            recommender.setTool(exportedRecommender.getTool());
+            recommender.setTraits(exportedRecommender.getTraits());
+
+            String layerName = exportedRecommender.getLayerName();
+            AnnotationLayer layer = annotationService.getLayer(layerName, aProject);
+            recommender.setLayer(layer);
+
+            recommender.setProject(aProject);
+            recommendationService.createOrUpdateRecommender(recommender);
+        }
+
+        int n = recommenders.length;
+        LOG.info("Imported [{}] recommender for project [{}]", n, aProject.getName());
```

Comment:
"recommenders" instead of "recommender" in message?

Reply:
Yes